src/Data/Food/Builder/Query.elm

```diff
@@ -92,25 +93,29 @@ carrotCake =
           , mass = Mass.grams 120
           , variant = Default
           , country = Nothing
+            , byPlane = False
```

Comment:
Is it expected indentation isn't consistant here?

Reply:
no idea how that happened, fixing this 👍